src/resmom/mom_comm.c

```diff
@@ -110,7 +110,7 @@ extern	int		server_stream;
 extern	char		mom_short_name[];
 extern unsigned int	pbs_mom_port;
 extern unsigned int	pbs_rm_port;
-extern int secondary_pbs_nodefile;
+extern int 		gen_nodefile_on_sister_mom;
```

Comment:
per our updated coding standards, variables and declarations need not be aligned - so you need to remove the alignment of all the blocks u touch